_js/go.mod

```diff
@@ -8,5 +8,6 @@ replace mvdan.cc/sh/v3 => ../
 
 require (
 	github.com/gopherjs/gopherjs v0.0.0-00010101000000-000000000000
+	golang.org/dl v0.0.0-20200811212135-d149fc5456ff // indirect
```

Comment:
undo this, and all the go.sum changes.

Reply:
OK, it should be related to `go get golang.org/dl/go1.12`.